include/zephyr/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -210,6 +210,7 @@ SECTIONS
 	__rodata_region_end = .;
 	MPU_ALIGN(__rodata_region_end - ADDR(rom_start));
 	__rom_region_end = .;
+	__rom_region_end = __rom_region_start + . - ADDR(rom_start);
```

Comment:
This would be nice to have for the linker script generator.

iirc this isn't directly supported today, the linker script generator and also armclang is still in progress, but I believe the use-case in this important to support also.

For now, no changes request in this PR on this part.